distribution/docs/src/main/resources/content/_managing/_configuring/environment-hardening.adoc

```diff
@@ -102,4 +102,29 @@ a|Update the Session configuration to have no greater than a 10 minute Session T
  * Select the *Configuration* tab. +
  * Select `Session`. +
  * Set `Session Timeout (in minutes)` to `10` (or less). +
+
+|Shell command access
+|TODO
+a|By default, access to some shell commands have been disabled due to how dangerous they can be.
```

Comment:
✏️ Something about "how dangerous they can be" sounds off to me, but I'm not sure how to reword this.
Maybe, "By default, some shell commands are disabled in order to secure the system." Other reviewers might have better thoughts on this.

Reply:
How Vina worded it sounds good to me 👍